src/components/CreateForm/index.js

```diff
@@ -0,0 +1,61 @@
+import {useState} from 'react';
+
+import useStore from '../../hooks/useStore.js';
+import Button from '../Button/formbutton.js';
+
+import FormContainer from './styled.js';
+
+export default function CreateForm() {
+	const [title, setTitle] = useState('');
+	const [category, setCategory] = useState('');
+	const [isChecked, setChecked] = useState(true);
```

Comment:
Hi Laura, you use two different states for one Object, I would combine them into one state like this: const [newMeal, setNewMeal] = useState(
{
tittle: '',
category: ''
}
)

Reply:
Thank you, Mathias, for your review. :)
I combined the two states now into one.